controller/config_test.go

```diff
@@ -203,22 +204,39 @@ func (s *ConfigSuite) TestAuditLogDefaults(c *gc.C) {
 	c.Assert(cfg.AuditLogCaptureArgs(), gc.Equals, false)
 	c.Assert(cfg.AuditLogMaxSizeMB(), gc.Equals, 300)
 	c.Assert(cfg.AuditLogMaxBackups(), gc.Equals, 10)
+	c.Assert(cfg.AuditLogExcludeMethods(), gc.DeepEquals, set.NewStrings("Client.FullStatus"))
 }
 
 func (s *ConfigSuite) TestAuditLogValues(c *gc.C) {
 	cfg, err := controller.NewConfig(
 		testing.ControllerTag.Id(),
 		testing.CACert,
 		map[string]interface{}{
-			"auditing-enabled":       true,
-			"audit-log-capture-args": true,
-			"audit-log-max-size":     "100M",
-			"audit-log-max-backups":  10.0,
+			"auditing-enabled":          true,
+			"audit-log-capture-args":    true,
+			"audit-log-max-size":        "100M",
+			"audit-log-max-backups":     10.0,
+			"audit-log-exclude-methods": []string{"Fleet.Foxes", "King.Gizzard"},
 		},
 	)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(cfg.AuditingEnabled(), gc.Equals, true)
 	c.Assert(cfg.AuditLogCaptureArgs(), gc.Equals, true)
 	c.Assert(cfg.AuditLogMaxSizeMB(), gc.Equals, 100)
 	c.Assert(cfg.AuditLogMaxBackups(), gc.Equals, 10)
+	c.Assert(cfg.AuditLogExcludeMethods(), gc.DeepEquals, set.NewStrings(
+		"Fleet.Foxes",
+		"King.Gizzard",
+	))
+}
+
+func (s *ConfigSuite) TestAuditLogExcludeMethodsType(c *gc.C) {
```

Comment:
I think we need validation and a test that the values are of the form <Facade>.<Method>

Reply:
I'm guessing this said `<facade>.<method>` before markdown mangled it - good call, I'll add that.